### PR TITLE
Add link for pre-built releases. [QUARKS-46]

### PR DESCRIPTION
**Pull request by cmdx**

### CODE REVIEW

File: site/docs/quarks-getting-started.md [hunk 1]
```diff
@@ -25,9 +25,13 @@ Quarks' primary API is functional where streams are sourced, transformed, analyz
 
 ### Downloading Apache Quarks
 
-To use Quarks, access the source code and build it. You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+To use Quarks, you can either download the source code and build it yourself, or you can download an install a pre-built release.
```
Comment (queeniema) [inline site/docs/quarks-getting-started.md]: Typo: "download _an_ install" should be "download _and_ install"

File: site/docs/quarks-getting-started.md [hunk 2]
```diff
@@ -25,9 +25,13 @@ Quarks' primary API is functional where streams are sourced, transformed, analyz
 
 ### Downloading Apache Quarks
 
-To use Quarks, access the source code and build it. You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+To use Quarks, you can either download the source code and build it yourself, or you can download an install a pre-built release. 
 
-After you build the Quarks package, you can set up your environment.
+You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+
+You can find pre-built releases [here](https://github.com/quarks-edge/quarks/releases).
```
Comment (queeniema) [inline site/docs/quarks-getting-started.md:32]: I think it would be a good idea to add a note here that these are not Apache releases.

**Comment by queeniema** [comment]
Hi @cmdx, thank you for your contribution! These edits look good :)

It looks like this PR is for [QUARKS-46](https://issues.apache.org/jira/browse/QUARKS-46?jql=project%20%3D%20QUARKS%20AND%20resolution%20%3D%20Unresolved%20AND%20component%20%3D%20Documentation%20ORDER%20BY%20priority%20DESC). Please include the JIRA issue number to your PR title so that it will get linked to the issue.

**Comment by cmdx** [comment]
Correct spelling mistake and add JIRA to title.

File: site/docs/quarks-getting-started.md (hunk 2 above)
Comment (queeniema) [inline site/docs/quarks-getting-started.md:32]: @cmdx Please make it explicit that the releases are not at Apache. Thank you!

**Comment by cmdx** [comment]
Add note stating that pre-built releases are not hosted at Apache.

File: site/docs/quarks-getting-started.md [hunk 3]
```diff
@@ -25,9 +25,14 @@ Quarks' primary API is functional where streams are sourced, transformed, analyz
...
 ### Downloading Apache Quarks
 
-To use Quarks, access the source code and build it. You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+To use Quarks, you can either download the source code and build it yourself, or you can download and install a pre-built release. 
 
-After you build the Quarks package, you can set up your environment.
+You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+
+You can find pre-built releases [here](https://github.com/quarks-edge/quarks/releases).
+Please note:  While Apache Quarks is undergoing incubation at The Apache Software Foundation (ASF), these pre-release builds will be hosted directly on gitub, not at Apache.
```
Comment (queeniema) [inline site/docs/quarks-getting-started.md:33]: Looks good @cmdx, but please fix the spelling error: _gitub_ to _GitHub_

Comment (ddebrunner) [inline site/docs/quarks-getting-started.md:33]: I think it needs to be noted that these are not Apache releases, where they are hosted is not really relevant.

**Pull request closed without merging**